frontend-ssr/components/cli_git/GitCli_CC.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import {uuid} from '../../services/Utils';
+
+import styles from './GitCli.module.scss';
+
+interface Props {
+  commands: RegExp[] | string[],
+  successMessages: string[],
+}
+
+interface State {
+  currentIndex: number,
+  messages: Message[],
+  btnMessage: string,
+}
+
+interface Message {
+  text: string,
+  type: MessageType,
+  time: string,
+  id: string,
+}
+
+enum MessageType {
+  COMMAND = 'command',
+  ERROR = 'error',
+  SUCCESS = 'success'
+}
+
+export default class MatchCommandClassComponent extends React.Component<Props, State, Message> {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentIndex: 0,
+      messages: [],
+      btnMessage:  'Next',
+    };
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+
+    const formEl = event.target;
+    const gitCommand = event.target.gitCommand.value;
+    const { commands, successMessages } = this.props;
+    const {currentIndex} = this.state;
+
+    let isMatch = false;
+
+    if (commands[currentIndex] instanceof RegExp) {
+      isMatch = (commands[currentIndex] as RegExp).test(gitCommand);
+    } else {
+      isMatch = commands[currentIndex] === gitCommand.trim().toLowerCase();
+    }
+
+    const date = new Date();
+
+    const commandMessage: Message = {
+      text: gitCommand,
+      type: MessageType.COMMAND,
+      time: date.toLocaleTimeString(),
```

Comment:
Let's not convert it to string here, but in the render function. Maybe later we want to do some checks/verificatins regarding those timestamps, and it will be easier if they are numbers.

So let's use: `time: Date.now()`